### PR TITLE
httpx: initial rerender

### PR DESCRIPTION
**Pull request by ke-zhang-rd**
…da-forge-pinning 2019.09.08

This meta.yaml is from https://github.com/conda-forge/httpx-feedstock/tree/master/recipe

### CODE REVIEW

File: recipe/meta.yaml [hunk 1]
```diff
@@ -17,28 +17,26 @@ build:
...
 test:
   imports:
     - httpx
-    - httpx/_backends
-    - httpx/_dispatch
   requires:
-    - pytest
+    - pip
   commands:
-    - pytest --pyargs sixtools.tests
```
Comment (mrakitin) [inline recipe/meta.yaml:41]: What was wrong with `pytest`?

Comment (ke-zhang-rd) [reply recipe/meta.yaml:41]: From here https://github.com/conda-forge/httpx-feedstock/tree/master/recipe

Before I use the meta.yaml from pypi. But it has some error in CI, now, this one is from condda-forge.

Comment (mrakitin) [reply recipe/meta.yaml:41]: What about the removed pytest command?

Comment (mrakitin) [reply recipe/meta.yaml:41]: Ah, I see, the pytest is not a part of the recipe on CF. Looks good.

**Pull request merged**